tests/Functional/WorkflowCommandsTest.php

```diff
@@ -109,6 +112,26 @@ public function testWorkflowOperationsAndLogsCommands()
         );
     }
 
+    /**
+     * testWorkflowWaitForCommitCommand
+     *
+     * @test
+     * @covers \Pantheon\Terminus\Commands\Workflow\Info\WaitForCommitCommand
+     * @group workflow
+     * @group short
+     * @throws GitException
+     */
+    public function testWorkflowWaitForCommitCommand()
+    {
+        $command = "vendor/bin/robo generate:test-commit";
```

Comment:
The Robofile is great for prototyping and ad-hoc usage, but I'd copy the code from the test-commit command into your test setup rather than exec-ing Robo here. Let's get everything else working before tackling that refactor, though.

Reply:
n.b. Terminus requires Robo, so you can use Robo collections and utility classes in your tests if you want.